packages/functions/lib/namespaced.ts

```diff
@@ -90,7 +173,7 @@ class FirebaseFunctionsModule extends FirebaseModule {
       }
     }
 
-    return data => {
+    return (data?: any) => {
```

Comment:
can this be improved.

Reply:
I don't think this can be really. It's up to the users discretion from what is returned from the function